komga-webui/src/locales/en.json

```diff
@@ -572,13 +572,13 @@
   "library_navigation": {
     "browse": "Browse",
     "collections": "Collections",
-    "readlists": "Read Lists",
+    "readlists": "Reading Lists",
     "recommended": "Recommended"
   },
   "login": {
     "create_user_account": "Create user account",
-    "login": "Sign in",
```

Comment:
Contention

Reply:
?

Reply:
Convention*. Was tired.

Reply:
Google, LinkedIn, Twitter and Github all use `Sign in`, which echoes to `Sign up`, while you can't say `Log up`. I will stick with `Sign in`.

Reply:
But you can "Register".